docs/install/Knative-custom-install.md

```diff
@@ -283,6 +283,15 @@ commands below.
    kubectl delete deploy knative-ingressgateway -n istio-system
    ```
 
+   You will also need to delete the following resource:
```

Comment:
This is exclusively a problem for people upgrading Eventing? I would call that out here, since not everyone uses all the components.

Reply:
Could just start the sentence with, "If you want to upgrade the Knative Eventing component..."

Reply:
Oh, Grant beat me to this suggestion above.